mobile/src/employee-details/employee-details-events-list.tsx

```diff
@@ -43,9 +45,29 @@ export class EmployeeDetailsEventsList extends Component<EmployeeDetailsEventsLi
                     <View style={eventRow}>
                         <ApplicationIcon name={this.eventTypeToGlyphIcon.get(item.type)} style={eventIcon} />
                         <StyledText style={eventTitle}>{item.type} starts on {item.dates.startDate.format(eventDialogTextDateFormat)} and completes on {item.dates.endDate.format(eventDialogTextDateFormat)} ({item.status})</StyledText>
+                        {
+                            this.eventManagementControls(this.props.events.find(e => e.calendarEventId === item.calendarEventId))
+                        }
                     </View>
                 </View>
         );
     }
 
+    private eventManagementControls(calendarEvent: CalendarEvent) {
+        return (calendarEvent.status === CalendarEventStatus.Requested) ? 
+        <View style={{ paddingLeft: 10, flexDirection: 'column', alignItems: 'center' }}>
```

Comment:
Move styles

Reply:
It's not final design - will be re-worked as app's mockups updated.